tests/manual_test_dydx.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2019 grandizzy
```

Comment:
Correct author?  Seems silly that we have a header in any of these `manual_test`s regardless.